gridt/src/app/login/register/register.page.spec.ts

```diff
@@ -11,7 +11,15 @@ describe("RegisterPage", () => {
   //testing suite, (de pagina te testen)
   let component: RegisterPage;
   let fixture: ComponentFixture<RegisterPage>;
-  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", ["create", "dismiss"]);
+  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", {
+    "create": {
+      "present": function() {
+        return true
+      }},
+    "dismiss": function() {
+      return true
```

Comment:
This seems to solve the issue for me as i ran the tests ten times and none of them failed.